vet.sh

```diff
@@ -41,6 +41,11 @@ if [[ "$1" = "-install" ]]; then
   popd
   if [[ -z "${VET_SKIP_PROTO}" ]]; then
     if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
+      if [[ -n "${VET_BUILD_GO_PROTOC}" ]]; then
+        pushd cmd/protoc-gen-go-grpc
+        go build -o /home/runner/go/bin .
+        popd
+      fi
```

Comment:
It's not evident to me why we need to build `protoc-gen-go-grpc` as part of tests/vet. Am I missing something here?

Reply:
Heey @arvindbr8, I had this issue without using that: 
https://github.com/grpc/grpc-go/actions/runs/7776819179/job/21204407334, 

its pipeline that was running without updated `vet.sh` to this change
```
 protoc-gen-go-grpc: program not found or is not executable
        Please specify a program using absolute path or make sure the program is available
```

even tho I have tried to add this in path using export, 
nothing seems to be good, and in `vet.sh` protoc is not being build to binary anywhere,
so only after this update I made this to be good

Reply:
Thanks for the clarification.